source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -774,7 +798,7 @@ export class ActionProperty extends PropertyDefinition {
     }
 
     toJSON(
-        _sender: SerializableObject,
```

Comment:
Unless we don't want to remove the "_" prefixes (which Paul told me were added because of the `noUnusedParameters` compiler directive, which doesn't seem to be applied anyway) then can you please avoid me the trouble or reverting those changes and opening another PR? Those name changes simply revert to how things used to be and have zero impact on how the code works. They're easily ignorable while reviewing.